File: tunnel-server/src/main/java/com/tuuzed/tunnel/server/internal/AttributeKeys.java
package com.tuuzed.tunnel.server.internal;

import io.netty.util.AttributeKey;
import org.jetbrains.annotations.NotNull;

public final class AttributeKeys {

    @NotNull
    public static final AttributeKey<Long> SESSION_TOKEN = AttributeKey.newInstance("$SESSION_TOKEN");
    @NotNull
    public static final AttributeKey<String> VHOST = AttributeKey.newInstance("$VHOST");
    @NotNull
    public static final AttributeKey<ServerTunnelSessions> SERVER_TUNNEL_SESSIONS
        = AttributeKey.newInstance("$SERVER_TUNNEL_SESSIONS");


}

File: tunnel2-server/src/main/kotlin/tunnel2/server/internal/_AttributeKeys.kt
@file:JvmName("AttributeKeys")
@file:Suppress("HasPlatformType")

package tunnel2.server.internal

import io.netty.util.AttributeKey

val AK_SESSION_ID = AttributeKey.newInstance<Long>("AK_SESSION_ID")
val AK_VHOST = AttributeKey.newInstance<String>("AK_VHOST")
val AK_PASSED = AttributeKey.newInstance<Boolean>("AK_PASSED")
val AK_SERVER_SESSION_CHANNELS = AttributeKey.newInstance<ServerSessionChannels>("AK_SERVER_SESSION_CHANNELS")

